.github/workflows/unit_tests.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Install HNN-core
         shell: bash -el {0}
         run: |
-          pip install '.[gui]'
+          pip install --verbose '.[gui]'
```

Comment:
just for debugging purposes?

Reply:
yeah, but I would keep it in so it's easier to debug if a CI fails in the future.